tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "module": "commonjs",
+    "rootDir": "./",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true
+  },
+  "files": []
```

Comment:
should we include the following fields?:
* `exclude`: we want to exclude the `node_modules` directory
* `allowJS`: since our entire project is TS this config wouldn't be compatible with our project

Reply:
Made the requested changes, kindly review it.

Reply:
@Aditya062003 looking great - just wanted to confirm: did you test these changes locally by running the tsc command?

Reply:
That would be the perfect way to test this change! There wouldn't be any new Jest test added, just convert app.js to app.ts!